pkgs/tools/video/rav1e/default.nix

```diff
@@ -13,17 +13,19 @@ in rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "sha256-V9QbztkFj3t5yBV+yySysDy3Q6IUY4gNzBL8h23aEg4=";
 
+  auditable = true; # TODO: remove when this is the default
+
   nativeBuildInputs = [ nasm cargo-c ];
   buildInputs = lib.optionals stdenv.isDarwin [ libiconv ];
 
   checkType = "debug";
 
   postBuild = ''
-    cargo cbuild --release --frozen --prefix=${placeholder "out"} --target ${rustTargetPlatformSpec}
+    $cargoCommand cbuild --release --frozen --prefix=${placeholder "out"} --target ${rustTargetPlatformSpec}
```

Comment:
Doesn't need to happen in this PR but any way we could avoid needing to do this? i.e wrap `cargo-auditable` as `cargo` or something like that?

Reply:
How do we stop the wrapper from recursively calling itself? Looks like cargo-auditable just calls `cargo` directly

https://github.com/rust-secure-code/cargo-auditable/blob/6275e86f503d9419b1528aa38af128882a648ba4/cargo-auditable/src/cargo_auditable.rs#L38

Reply:
Maybe we can look at `CARGO_AUDITABLE_ORIG_ARGS`, but I'm not sure if we should rely on that since it is internal and undocumented

Reply:
Another way to do it is just to add something like a `cargoAuditableHook` that sets `RUSTC_WORKSPACE_WRAPPER`

Reply:
> How do we stop the wrapper from recursively calling itself? Looks like cargo-auditable just calls `cargo` directly

Make `cargo` into `cargo-unwrapped`, substitute `cargo` for `cargo-unwrapped` in `cargo-auditable` and then wrap it?

Reply:
> Make `cargo` into `cargo-unwrapped`

Sounds like the most future-proof way of doing it, but it would complicate the bootstrap process a bit.

Due to the complexity of this, I think it is worth a separate issue once this is merged

Reply:
> Due to the complexity of this, I think it is worth a separate issue once this is merged

That's fine but I do think it's something that should be resolved before `auditable` is made the default.

`$cargoCommand` in an "extra" step like it is here isn't too bad but much easier for everyone if people don't need to know about it.

Should also mean that it works without changes for packages that have cmake, make, meson, etc build systems calling `cargo`.

Reply:
Hey! `cargo auditable` author here 👋 

I'll see what I can do about providing a robust way of using `cargo auditable` as a drop-in replacement for `cargo.`

Here's the upstream issue so you can track the progress: https://github.com/rust-secure-code/cargo-auditable/issues/89

Reply:
@Shnatsel thanks for all the work on cargo-auditable.
It works! I wasn't able to use an alias since `cargoBuildHook` uses `env`, which doesn't pick up aliases, but a shell script wrapper does the job